x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/dataframe/action/GetDataFrameTransformsAction.java

```diff
@@ -49,33 +46,24 @@ public Response newResponse() {
         return new Response();
     }
 
-    public static class Request extends ActionRequest implements ToXContent {
-        private String id;
+    public static class Request extends AbstractGetResourcesRequest implements ToXContent {
+
+        public static final String ALLOW_NO_TRANSFORMS = "allow_no_transforms";
```

Comment:
seems not to be used in this class but only in `RestGetDataFrameTransformsAction`, so I would move it there, except you plan to use it here.